src/main.py

```diff
@@ -14,7 +14,8 @@ async def main():
     Resources must be pre-registered. For an example, see the `__init__.py` file.
     """
     module = Module.from_args()
-    module.add_model_from_registry(Camera.SUBTYPE, OakDModel.MODEL)
+    for model in Oak.MODELS:
```

Comment:
nice! Are we certain that this driver will expand to all oak hardware? 

You can totally change the tuple to oak_seomthing and use this, special casing the other models that need new constructors and validators.

Reply:
Yeah I think so! The DepthAI docs are pretty clear that it will support all OAK cameras, so as long as their SDK can do it, we can do it. https://docs.luxonis.com/en/latest/